requirements.txt

```diff
@@ -0,0 +1,15 @@
+-c constraints.txt
+
+Sphinx
+jsx-lexer
+lesscpy
+linkify-it-py
```

Comment:
I don't recognize the above two requirements for Sphinx. Should we have separate requirements files, one for the app, and the other for docs?

Reply:
I will copy over the documentation repo one.